core/unit_test/TestTeamMDRange.hpp

```diff
@@ -1432,8 +1432,8 @@ struct TestThreadVectorMDRangeParallelReduce : public TestTeamMDParallelReduce {
                   vectorSum += v(leagueRank, i, j, k);
                 },
                 threadSum);
-
-            teamSum += threadSum;
+            Kokkos::single(Kokkos::PerThread(team),
+                           [&]() { teamSum += threadSum; });
```

Comment:
Just add to leagueSum in the single. There is really zero value to the teamSum temporary.

Reply:
I agree. Addressed in 2c9b873.